zeek-client

```diff
@@ -596,6 +612,43 @@ class Node(BrokerType, ConfigParserMixin):
                 self.cpu_affinity,
                 hdenv)
 
+    def to_json_data(self):
+        return {
+            'name': self.name,
+            'instance': self.instance,
+            'role': self.role.to_json_data(),
+            'state': self.state.to_json_data(),
```

Comment:
I'vm wondering about the meaning of 'state' from a user perspective: having that in `INI`, suggests that it's something the user can change as well. Is that indeed how it's meant? 

It all makes sense on the `get-config` side to understand the current state of the cluster. I guess I'm wondering about `set-config` and if this is a supported version of controlling nodes (kind on top of `start`, `stop` etc..

(Not sure this really matters much in the end, just trying to understand semantics).

Reply:
Yeah, I was pondering that one too. The only other state that I could see a user care about right now is `PENDING`, which would convey the identity of a cluster node but not immediately run it. (This would still need to be implemented, though.) However, this would assume the ability to adjust the Zeek cluster at runtime, which doesn't work until we teach the cluster framework to become more dynamic. Otherwise you need cluster-wide restarts anyway, and we're back to implying state `RUNNING`.

I think I'll remove the node state from the user-visible configs for now. (It remains there under the hood since the `state` field is part of the `Node` data structure.) We can expose it in the future when that makes sense. In the meantime the user gets the current node state via `get-nodes`, and can start/stop/restart on demand via the upcoming set of commands.